index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link
+  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+  rel="stylesheet"
+/>
+</head>
+<body>
+  <header>
+      <nav>
+          <div class="logo-part">
+              <h4 class="logo-text">My Logo</h4>
```

Comment:
- [ ] I believe this was correct before as the logo requires an anchor `a` tag around it so that visitors can quickly navigate to the home page just by clicking on the logo. Kindly put it back to make it a user-friendly design 😃